backend/routes/index.ts

```diff
@@ -1,11 +1,13 @@
 import express from "express";
 import FilesController from "../controllers/FilesController";
+import UserController from "../controllers/UserController";
 import LocationsController from "../controllers/LocationsController";
 import AuthController from "../controllers/AuthController";
 import passport from "../libs/passport";
 
 const router = express.Router();
 
+router.get("/userData/:userToken", UserController.getUserData);
```

Comment:
req.params.userToken. instead, better use _id, and find by it. I don't understand the purpose of token here, as it's revealed in browser query (which is not secure).